test/vbinary-microkernel-tester.h

```diff
@@ -161,7 +199,12 @@ class VBinaryMicrokernelTester {
   uint8_t a_zero_point_{121};
   uint8_t b_zero_point_{127};
   uint8_t y_zero_point_{133};
+  int16_t a_zero_point_s16_{121};
```

Comment:
I think these constants are too small, they don't exercise a lot of the range that is valid for this op. There should be negative zero points, and zero points closer in magnitude to `2^15 - 1`.

Reply:
changed to magnitude closer to int16 range